spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'PostsController', type: :request do
+  describe 'GET posts' do
+    it "should render 'index.html.erb' view" do
+      get '/users/1/posts'
+      expect(response.body).to include('list of posts for a given user')
+    end
+
+    it 'should render users/1/posts/index' do
+      get '/users/1/posts'
+      expect(response).to have_http_status(200)
+      expect(response).to render_template('index')
+    end
+  end
+
+  describe 'Get post/1' do
+    it "should render 'show.html.erb' view" do
+      get '/users/1/posts/1'
+      expect(response.body).to include('Get post by id')
+    end
+
+    it "should render 'show.html.erb' view" do
+      get '/users/1/posts/1'
+      expect(response.body).to render_template('show')
+    end
+  end
```

Comment:
- [ ] Your post show tests are failing. Kindly fix them.

  ![post-show](https://user-images.githubusercontent.com/36542986/179816102-d73fdd5f-e6d9-4be3-a0ee-1a8b145124d2.jpg)

Reply:
@TedLivist thanks for the review and i tried to run the test again all are working 
Note: incase of next reviewer if there is a fail case kindly run by line I.e  rspec ./spec/requests/users_spec.rb: 7 to help me locate the fail test thanks all.


![Screenshot (191)](https://user-images.githubusercontent.com/37457094/179827597-e7fd0650-5bf7-402b-8633-a5c7f9fe491f.png)